docs/source/hi/_toctree.yml

```diff
@@ -0,0 +1,79 @@
+    - local: installation
+      title: (इंस्टालेशन) Installation
+- title: "कैसे करें मार्गदर्शन"
+  sections:
+    - local: guides/overview
+      title: (निरीक्षण) Overview
+    - local: guides/download
+      title: (फ़ाइलें डाउनलोड करें) Download files
+    - local: guides/upload
+      title: (फाइलें अपलोड करें) Upload files
+    - local: guides/hf_file_system
+      title: (एचएफफ़ाइलसिस्टम) HfFileSystem
+    - local: guides/repository
+      title: (कोष) Repository
+    - local: guides/search
+      title: (तलाश) Search
+    - local: guides/inference
+      title: (अनुमान) Inference
+    - local: guides/community
+      title: (समुदाय टैब) Community Tab
+    - local: guides/collections
+      title: (संग्रह) Collections
+    - local: guides/manage-cache
+      title: (द्रुतिका) Cache
+    - local: guides/model-cards
+      title: (मॉडल कार्ड) Model Cards
+    - local: guides/manage-spaces
+      title: (अपना स्पेस प्रबंधित करें) Manage your Space
+    - local: guides/integrations
+      title: (एक library एकीकृत करें) Integrate a library
+    - local: guides/webhooks_server
+      title: (वेबहुक सर्वर) Webhooks server
+- title: "वैचारिक मार्गदर्शक"
+  sections:
+    - local: concepts/git_vs_http
+      title: Git बनाम HTTP प्रतिमान
+- title: "संदर्भ"
+  sections:
+    - local: package_reference/overview
+      title: (अवलोकन) Overview
+    - local: package_reference/login
+      title: (लॉगइन करें और लॉगआउट करें) Login and logout
+    - local: package_reference/environment_variables
+      title: (पर्यावरण चर) Environment variables
+    - local: package_reference/repository
+      title: (स्थानीय और ऑनलाइन रिपॉजिटरी का प्रबंधन करना) Managing local and online repositories
+    - local: package_reference/hf_api
+      title: (हगिंग फेस हब एपीआई) Hugging Face Hub API
+    - local: package_reference/file_download
+      title: (फ़ाइलें डाउनलोड हो रही हैं) Downloading files
+    - local: package_reference/mixins
+      title: (मिश्रण और क्रमांकन विधियाँ) Mixins & serialization methods
+    - local: package_reference/inference_client
+      title: (अनुमान ग्राहक) Inference Client
+    - local: package_reference/hf_file_system
+      title: (एचएफफ़ाइलसिस्टम) HfFileSystem
+    - local: package_reference/utilities
+      title: (उपयोगिताओं) Utilities
+    - local: package_reference/community
+      title: (चर्चाएँ और अनुरोध खींचें) Discussions and Pull Requests
+    - local: package_reference/cache
+      title: (कैश-सिस्टम संदर्भ) Cache-system reference
+    - local: package_reference/cards
+      title: (रेपो कार्ड और रेपो कार्ड डेटा) Repo Cards and Repo Card Data
+    - local: package_reference/space_runtime
+      title: (स्पेस रनटाइम) Space runtime
+    - local: package_reference/collections
+      title: (संग्रह) Collections
+    - local: package_reference/tensorboard
+      title: (टेंसरबोर्ड logger) TensorBoard logger
+    - local: package_reference/webhooks_server
+      title: (वेबहुक सर्वर) Webhooks server
```

Comment:
```suggestion
```

I would remove sections that are not translated yet so that users don't have expectations when seen them. It will not brake any existing links/urls as the user will be redirected automatically to the English version in case they click on a link that doesn't exist yet in Hindi. It is not a problem to have only a few links on the sidebar at the beginning for a language that is currently been translated.

I would also remove the guides written in English from this PR.

Reply:
Removed the Hindi translation for the non-translated guides or directories.Is it suggestable that this file be deleted?? As the translation I am doing are based on 3 files that are not mentioned in the _toctree.yml file.

Reply:
@aneeshd27  In my opinion, the doctree should look like this:

```yaml
- title: "Get started"
  sections:
    - local: index
      title: Home
    - local: quick-start
      title: Quickstart
    - local: installation
      title: Installation
```

but translated to Hindi and that's it. No references to the non-translated pages.